firmware/src/net/lwipopts.h

```diff
@@ -91,6 +90,11 @@
 #define SNTP_STARTUP_DELAY 0
 #define SNTP_COMP_ROUNDTRIP 1
 
+// Forward declarations for custom time functions defined in ntp.c
```

Comment:
With the `hardware/rtc.h` the ntp.h was getting pulled here due to the magic of C and that was allowing the functions to resolve here.
`ntp.h` pulls `time.h` & `pico/stdlib.h` and is somewhat problematic to include here, hence the forward declaration.